### ISSUE TITLE
додати нові слова - 16.03.2019

### ISSUE THREAD

**Issue by vstarko** [brown-uk/dict_uk#179]
ВЧ noun:p # військова частина; висока частота
ВРХ noun:m:nv:np # велика рогата худоба
ВС noun:m:nv:np:prop #Верховний суд; військовий стан
ВС noun:p:nv #внутрішніх справ; військові сили
ПМС noun:m:nv:np # передменструальний синдром
ІПСШ noun:p:nv # інфекції, які передаються статевим шляхом
Держлікслужба /n10 :np # Державна служба України з лікарських засобів та контролю за наркотиками
ГАМК noun:f:nv # гамма-аміномасляна кислота
прям adv:coll
акмеологічний /adj.adv
гамма-аміномасляний /adj
дозвіллєвий /adj
богопосвячений /adj :&adjp:pasv:perf    # :past
здоров'язбережувальний /adj
здоров'язберігаючий /adj :&adjp:actv:imperf #> здоров'язбережувальний
реколекція /n10.p1
матушка /n10.p2.ko.< :rare
тета /n10.p1.ko.< :xp1 :rare
тета /n10.p1.ko :xp2
інет /n20 :coll
проставлення /n2n.p1
духовість /n30.p
поскільки conj:subord #> оскільки
противсіх /n20.a.p.<
сайдинг /n20.p
дорадництво /n2n
диклофенак /n20
секонд /n20.p
таймс noninfl:foreign
фешн /n20
подивляти /v2.cf.advp :imperf
заметатися /vr1 /vr2 :perf 
бухтіти /v1.cf.advp :imperf
пресс noninfl («Ассошіейтед пресс, Юнайтед Пресс, Франс-пресс і Франс Пресс»)
імхо insert:slang
вуаля intj:&insert
рах-к /n22.p # рахунок 
Агро noninfl
Інформ noninfl 
Інвест, інвест noninfl (часто також у складі назв -інвест, -Інвест)
акаунт- (напр., акаунт-менеджер)
хард- (хард-рок)
інді- (інді-гурт)
-денс (стріт-денс)
-емерит (професор-емерит)
аш- (аш-Шаллад)
дозвіллєво-
тепло-, енерго- (якість тепло-, енерго- і водопостачання
клад #> скарб
одвідати
одпочити
пропагандувати
розглядіти
часть
приказ
сего
блін
ученик
ік (прийменник, :rare)
она (застаріле, :rare
си :rare – діал. частка ся
богато :rare – застаріле
бердичівлянин /n20.a.pyn.ke.<   #> бердичівець
бердичівлянка  /n10.p2.ko.<   #> бердичівка
Грубешів (Грубешова)
Назар /n20.a.p.ke.<.patr :xp1
Назар /n20.a.p.ke.<+ :xp2
Виготський /n2adj1.f.p.<+
Росс /n20.a.p.ke.<+ :xp1
Росс /n20.a.ke.< :xp2
Росс /<f :xp3
Чак /n20.a.p.<
Рут /<f
Пій /n20.a.p.<
Тан /n20.a.p.ke.<+ :хp1 чоловіче прізвище
Тан /<+ :xp2 жіноче прізвище; династія
Таймс /n20.a.u
Гіркін /n2adj2.<+m
Єлисавета /n10.p1.ko.<

**Comment by arysin** [comment]
Щодо цих маю сумніви, чи варто:
часть
приказ
сего
блін
ученик

**Comment by vstarko** [comment]
Це високочастотні слова в корпусі. Ми їх можемо додати в словник як :rare і/або :bad і подати заміну - на сучасному рівні це буде правильно. Водночас тоді ці слова тегуватимуться в старих текстах. Виняток: блін insert:&slang

**Comment by arysin** [comment]
гаразд, а які теги у «сего»? для нього треба лему «се»?

**Comment by vstarko** [comment]
Ні. У нас вже є лема се adj. Від неї утворено сього adj:n:v_rod:&pron:dem:rare
Такі самі теги треба для сего. Можна додати як окрему форму.
До речі, є ще значно менш частотне сему (=сьому), але його, мабуть, ліпше не чіпати.

**Comment by arysin** [comment]
Додав всі, крім цих (треба подумати, як найкраще це зробити):
пресс noninfl («Ассошіейтед пресс, Юнайтед Пресс, Франс-пресс і Франс Пресс»)
Агро noninfl
Інформ noninfl
Інвест, інвест noninfl (часто також у складі назв -інвест, -Інвест)
аш- (аш-Шаллад)
ік (прийменник, :rare)
она (застаріле, :rare
си :rare – діал. частка ся
богато :rare – застаріле
